coding/compressed_bit_vector.cpp

```diff
@@ -231,6 +231,14 @@ void DenseCBV::Serialize(Writer & writer) const
   rw::WriteVectorOfPOD(writer, m_bitGroups);
 }
 
+unique_ptr<CompressedBitVector> DenseCBV::Clone() const
+{
+  DenseCBV * cbv = new DenseCBV();
```

Comment:
I'm not sure it is possible: we need to access fields of the derived class which cannot be done via  a unique_ptr to its base class.